rosdep/python.yaml

```diff
@@ -4170,6 +4170,13 @@ python-responses-pip:
   ubuntu:
     pip:
       packages: [responses]
+python-rhino3dm:
```

Comment:
do you want to add the package for python3?

If that's the case, please add a `python3-rhino3dm-pip` key instead of `python-rhino3dm`.
(the -pip ending must be added if the package is provided by pip)

Please, read the contributing guidelines https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#python-rules.